cmd/stability/differ_test.go

```diff
@@ -0,0 +1,118 @@
+package stability
+
+import (
+	"io/ioutil"
+	"testing"
+
+	pb "github.com/apigee/registry/rpc"
+	"github.com/golang/protobuf/proto"
+)
+
+const (
+	expected_diff =
+			`info:
+    version:
+        from: 1.0.0
+        to: 1.0.1
+components:
+    schemas:
+        modified:
+            Pet:
+                required:
+                    deleted:
+                        - name
+                properties:
+                    added:
+                        - added
+                        - age
+`
+	expected_name_1 = "Test"
+	expected_name_2 = "Test"
```

Comment:
It's difficult to understand what these constants are used for given their current names. Can the names be more descriptive, or can they be defined in the tests instead so we have context to help understand?

Also, recall that names should be [camelCase or MixedCase](https://golang.org/doc/effective_go#mixed-caps). These tests have lots of underscores that make it harder to write the variable names.